spec/controllers/concerns/reauthentication_required_concern_spec.rb

```diff
@@ -1,19 +1,26 @@
 require 'rails_helper'
 
-describe ReauthnRequiredController do
+describe ReauthenticationRequiredConcern do
   let(:user) { create(:user, :signed_up, email: 'old_email@example.com') }
 
+  class ReauthenticationRequiredController < ApplicationController
+    include ReauthenticationRequiredConcern
+
+    before_action :confirm_recently_authenticated
+
+    def show
+      render plain: 'Hello'
+    end
+  end
```

Comment:
I think we can use the rails `controller` helper here, to avoid defining a class (I try to avoid defining specs in classes because they end up at the global namespace and sometimes collide with each other)

```suggestion
describe ReauthenticationRequiredConcern, type: :controller do
  let(:user) { create(:user, :signed_up, email: 'old_email@example.com') }

  controller do
    include ReauthenticationRequiredConcern

    before_action :confirm_recently_authenticated

    def show
      render plain: 'Hello'
    end
  end
```

Reply:
thanks, rspec required a few other changes that I compiled into 56d312658

Primary issue was:
```
superclass must be an instance of Class (given an instance of Module)
```